app/views/smart_answers/custom_result.html.erb

```diff
@@ -26,10 +26,14 @@
       "action": "complete",
       "tool_name": @presenter.title,
     }.to_json %>">
-    <%= render "govuk_publishing_components/components/title", {
-      title: outcome.title,
-      margin_bottom: 6,
-    } %>
+    <div class="govuk-!-margin-top-8">
+      <%= render "govuk_publishing_components/components/heading", {
+        text: outcome.title,
+        margin_bottom: 6,
```

Comment:
Interesting that this is value 6 here for bottom margin. I've used 8 for the ones I've done and there's been no visual difference. Does it look wrong if you set it to 8?